src/jdk.jfr/share/conf/jfr/default.jfc

```diff
@@ -512,6 +512,16 @@
       <setting name="cutoff" control="old-objects-cutoff">0 ns</setting>
     </event>
 
+    <event name="jdk.NativeMemoryUsage">
+      <setting name="enabled" control="gc-enabled-normal">true</setting>
```

Comment:
gc-enabled-normal? Should this really be tied to a gc control? An NMT control to be introduced?

Reply:
Since users need to pass a command line flag to enable the event, I think it can be on by default, and use included gc-enabled-normal

Reply:
Talked to @egahlin about this and maybe not optimal to tie it to a gc control, but fine for now. This does not affect the VM, just JMC.